### PR TITLE
add aws ses support

### PR DESCRIPTION
**Pull request by spartan2015**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -408,6 +408,41 @@ static:
         - Minimum
         period: 60
         length: 300
+  - namespace: AWS/SES
```
Comment (cristiangreco) [inline README.md:411]: Rather than this full example, maybe just add SES to the list of supported services at the top of the file.

Comment (cristiangreco) [reply README.md:411]: Ignore this comment, the service is already there. So this is just a fix to add the `ResourceFilters`?

Comment (marekhanzlik) [reply README.md:411]: Does the `ResourceFilter` affect this? 
https://github.com/nerdswords/yet-another-cloudwatch-exporter/issues/738

**Comment by cristiangreco** [comment]
Closing this PR as the change shouldn't be needed anymore since #738 has been fixed.

**Pull request closed without merging**